plugins/auth-backend/api-report.md

```diff
@@ -692,13 +651,15 @@ export const providers: Readonly<{
   }>;
 }>;
 
-// Warning: (ae-missing-release-tag) "readState" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
 // @public (undocumented)
 export const readState: (stateString: string) => OAuthState;
 
-// Warning: (ae-missing-release-tag) "RouterOptions" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
+// @public (undocumented)
+export type RedirectInfo = {
```

Comment:
Should probably be called `OAuthStartResponse` instead

Reply:
Do you want me to refactor to rename it to that? Considering it's OK since the actual symbol wasn't exported properly in the first place so it effectively doesn't change the "usable" interface right

Reply:
OK, added as afc50cb

Reply:
👍 , not a breaking change tho right?

Reply:
Shouldn't be I guess? You be the judge :) If I understood it correctly, it seems that you wouldn't have been able to import it properly from the outside.